back-end/src/sciler/handler/handler_helper.go

```diff
@@ -206,6 +250,22 @@ func (handler *Handler) getCameras() []map[string]string {
 	return cameras
 }
 
+// getButtons returns a list with button names
+func (handler *Handler) getButtons() []map[string]interface{} {
+	var buttons []map[string]interface{}
+	for _, btn := range handler.Config.ButtonEvents {
+		rule, ok := handler.Config.RuleMap[btn.ID]
+		if !ok {
+			logrus.Errorf("rule with id %s not found in rule map for button %s", rule.ID, btn.ID)
```

Comment:
is it possible to get to this error, and if so then it should have been checked reading the config in